src/wise_psf.jl

```diff
@@ -0,0 +1,39 @@
+using FITSIO
+
+function wise_psf(band; sidelen=-1)
+
+  # don't believe any sensible normalization is in place, need to fix that ...
+
+  # could potentially add checks on the input values ...
```

Comment:
Declaring the type of `band` to be an `Int64` in the function header probably suffices, i.e. `band::Int64`.